redwood-layout-shared-test/src/commonMain/kotlin/app/cash/redwood/layout/AbstractFlexContainerTest.kt

```diff
@@ -429,6 +451,8 @@ interface TestFlexContainer<T : Any> : Widget<T>, ChangeListener {
   fun mainAxisAlignment(mainAxisAlignment: MainAxisAlignment)
   fun margin(margin: Margin)
   fun add(widget: Widget<T>)
+  fun addAt(index: Int, widget: Widget<T>)
```

Comment:
The first thing I tried was exposing the `Widget.Children` list in this facet. That doesn’t work so well 'cause LazyList doesn’t know its own count until `onEndChanges()` is called.